src/dispatcher.js

```diff
@@ -140,7 +141,8 @@ export default class Dispatcher {
         }
 
         const cursor = this.selectionWatcher.getFreshSelection()
-        clipboard.paste(block, cursor, afterPaste)
+        const clipboardContent = evt.clipboardData.getData('text/html')
```

Comment:
Is `text/html` always present? What happens with paste using `CMD + SHIFT + v`?
In that case is it also present or do we need a fallback to `text/plain`?

Reply:
list of media types: https://www.w3.org/TR/clipboard-apis/#reading-from-clipboard

Reply:
Yes, very good point, with only `text/html`, nothing happened on Cmd + Shift + v, so I added a fallback for `text/plain`.